ci/test-conformance-aks.sh

```diff
@@ -25,14 +25,15 @@ K8S_VERSION="1.16.10"
 RESOURCE_GROUP="antrea-ci-rg"
 RUN_ALL=true
 RUN_SETUP_ONLY=false
+SETUP_TEST_ONLY=false
 RUN_CLEANUP_ONLY=false
 KUBECONFIG_PATH="$HOME/jenkins/out/aks"
 TEST_FAILURE=false
 MODE="report"
 
 _usage="Usage: $0 [--cluster-name <AKSClusterNameToUse>] [--kubeconfig <KubeconfigSavePath>] [--k8s-version <ClusterVersion>]\
                   [--azure-app-id <AppID>] [--azure-tenant-id <TenantID>] [--azure-password <Password>] \
-                  [--aks-region <Region>] [--log-mode <SonobuoyResultLogLevel>] [--setup-only] [--cleanup-only]
+                  [--aks-region <Region>] [--log-mode <SonobuoyResultLogLevel>] [--setup-only] [--setup-test-only] [--cleanup-only]
```

Comment:
Don't you think `--test-only` would be more useful, in case someone wants to run the tests multiple times?

Reply:
In my own experience I don't feel there is a need for that... The conformance tests usually take a while to complete whereas the cluster setup/teardown time is way less. If someone wants to run tests multiple times, they can setup cluster, test, tear down cluster and repeat, just like we do in Jenkins. Also I don't see a point of running the conformance tests on the same cluster multiple times if the first run succeeds... But I can totally see someone new to Antrea wants to play around in an antrea-enabled EKS cluster but do not wish to run the whole conformance tests. That's sort of the motivation behind the 'setup-only' option

Reply:
Sounds good. I do suggest renaming the flag `--setup-and-test-only` though (along with the corresponding variable).

Reply:
done